OffsetLib/en.py

```diff
@@ -0,0 +1,35 @@
+import ktl
+import argparse
+
+from wftel import wftel
+import KeckLogger
+
+parser = argparse.ArgumentParser(description="Move the telescope in the RA-DEC directions",
+                         usage="en.py raoff decoff")
+
+parser.add_argument("raoff", type = float, help="ra offset")
+
+parser.add_argument("decoff", type=float, help="dec offset")
+
+
+args = parser.parse_args()
+
+
+def en(x, y):
+    if(x == 0.0 and yf == 0.0):
```

Comment:
Equals `==` and floats are always a bit perilous.  Perhaps `if x < 0.001 and y < 0.001` or something like that?